src/scalems/radical/__init__.py

```diff
@@ -347,7 +348,7 @@ def _describe_task(item: _context.Task, scheduler: str) -> rp.TaskDescription:
     return task_description
 
 
-async def submit(item: _context.Task, task_manager: rp.TaskManager, submitted: asyncio.Event = None,
+async def submit(*, item: _context.Task, task_manager: rp.TaskManager, pre_exec: list, submitted: asyncio.Event = None,
```

Comment:
I've never seen this syntax before - what does it do?

Reply:
I assume you mean the `*`? In the same way that `*args` captures all remaining positional arguments into a tuple called `args`, a bare `*` simply marks the end of positional arguments, requiring explicit key word arguments for the following parameters. I used it here to start disallowing hard-to-maintain call sites as the argument list is growing.
Python 3.8 also adds `/` to mark the preceding parameters as positional-only, which can be irritating to users, but which allows for simpler logic when inspecting or manipulating call signatures, such as with functools.partial, functools.singledispatch, or when capturing `**kwargs`.
See also PEP-570. (I can never find this stuff in the library reference.)